_posts/2020-06-23-sourcify-faq.md

```diff
@@ -0,0 +1,195 @@
+on a third party which verifies the code for you. In that scenario, the verification service providers could be an attack 
+vector and would need to be trusted in.
+
+## Metadata 
+
+### What is bytecode metadata?
+
+Metadata is data that describes and gives information about other data. [Bytecode (contract) metadata](https://solidity.readthedocs.io/en/latest/metadata.html) 
+in Solidity is supported since version 0.4.7 and is automatically generated by the Solidity compiler in form of a JSON file. 
+The metadata file contains information about the compiled contract, e.g. the used compiler version, the sources used, 
+the ABI and the NatSpec documentation. The metadata file also contains IPFS and SWARM hashes of the source code, which allows 
+to retrieve them if they are available.
+
+Metadata files can also contain literal sources, which means that the acutal code of the contract is present in the metadata 
+file's source section. Literal sources will be covered by the Sourcify verification [in future](https://github.com/ethereum/sourcify/issues/187), too.
+
+### What is NatSpec?
+
+[NatSpec](https://solidity.readthedocs.io/en/latest/natspec-format.html) is short for Ethereum Natural Language Specification 
+Format. Solidity contracts can use NatSpec comments to provide rich documentation for functions, return variables and more. 
+The comments are categorized by developer-focused and end-user-focused messages. The end-user-focused messages can be crucial 
+to users since they describe what the contract interaction will do in easy language. It is highly recommended to fully 
+annotate Solidity contracts using NatSpec for all public interfaces. 
+
+NatSpec comments can be displayed to the user at the time of interaction, e.g. by being promted in the wallet interface 
+before signing a transaction.
+
+## Sourcify
+
+### What is Sourcify?
+
+Sourcify is an initiative which aims to make interacting with smart contracts on the blockchain safer and more transparent 
+for users. 
+
+To achieve this goal, Sourcify supports several efforts to foster adoption of open-source source verification, metadata files 
+and NatSpec comments.
+
+At its core, Sourcify currently maintains
+- an [interface](https://verification.komputing.org/) that helps developers to verify metadata and contract source code.
+- a [contract repository](https://contractrepo.komputing.org/) of all verified contracts.
+- a [monitoring & verifier service](https://github.com/ethereum/sourcify) that checks for new blocks on Ethereum blockchains 
+(mainnet and testnets) and tries to verify them automatically.
+
+Besides the technical infrastructure, Sourcify is also a collective movement to bring transparency and awareness to the space. 
+We try to educate and build bridges between development tools, wallets, interfaces and other components which all play an 
+important role in demystifying interaction with smart contracts for the end user and hence making blockchain interactions safer.
+
```

Comment:
Maybe stress here that sourcify only provides the base layer where other tools can build on top?

Maybe also stress that the main purpose is to keep metadata and sources available via ipfs and swarm so that the links in the bytecode do not turn into dead links.

Reply:
Will add your points.